exercises/sum-of-multiples/src/main/java/SumOfMultiples.java

```diff
@@ -1,3 +1,11 @@
-public class SumOfMultiples {
-    
+class SumOfMultiples {
+
+    SumOfMultiples(int number, int[] set) {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
+    }
+
+    int getSum() {
+        throw new UnsupportedOperationException("Delete this statement and write your own implementation.");
+    }
+
```

Comment:
I suggest making the constructor and the method `public`. Although leaving them out won't cause problems for running the tests (since they are in the same default package), that's not generally how they would be written, and I be